.ci/dockerfiles/Dockerfile.base

```
@@ -32,6 +32,18 @@ RUN apt-get update && \
     apt-get install -y sudo python3 python3-pip kmod && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
+# Google Cloud SDK (gcloud + gsutil) for fetching closed-source DDN Infinia
+# libraries from a private GCS bucket at CI runtime (see .gitlab/fetch_infinia.sh).
+RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)